docs/organization-integration/onpremise-integration.md

```diff
@@ -0,0 +1,173 @@
+- Authentication is based on your identity provider, using [OpenID Connect](https://openid.net/developers/how-connect-works/). 
+- Analyses are triggered from a build pipeline. The analysis results are then imported into a Postgres database, so they can be viewed in Sigrid.
+- Large files are stored in an [S3-compatible object store](https://aws.amazon.com/s3/).
+
+Some Sigrid on-premise features are *optional* based on your environment:
+
+- The Open Source Health feature requires outbound internet access, to retrieve the latest version of all known vulnerabilities in open source libraries. 
+- When viewing detailed analysis results, Sigrid displays relevant source code files within Sigrid. For this to work, a web-accessible code storage needs to be available. This integrates with Sigrid via [OAuth](https://oauth.net/2/). For this to work, the identity provider used for Sigrid authentication and for the code storage needs to be the same.
+
+### Environment requirements
+
+- Your infrastructure needs to support running applications on Kubernetes.
+- You need to pull Sigrid container images from DockerHub.
+  - If you do not allow outbound internet access, a custom air-gapped installation and update procedure needs to be designed, documented and followed.
+- You need to support and manage a Postgres database service.
+- You need to have a development platform with a continuous integration pipeline, since Sigrid will integrate with this pipeline. Supported development platforms are listed in the "Sigrid CI pipeline integration" in this documentation.
+- You need to have an identity provider supporting OpenID Connect, since Sigrid will integrate with this for authentication.
+- You need to allow outbound internet traffic in order to support Sigrid’s Open Source Health feature. Sigrid needs to connect to external sources to check for the latest vulnerability data.
+  - If you do not allow outbound internet access, the Open Source Health feature is not available. The rest of Sigrid is unaffected.
+- For viewing source code within Sigrid, you need to provide a development platform that is integrated with th esame identity provider as Sigrid itself.
+  - The view source functionality is optional, without this integration the rest of Sigrid is unaffected.
+
+## Obtaining Sigrid on-premise
+
+The Docker containers that form Sigrid on-premise are distributed via [DockerHub](https://hub.docker.com). You will receive an account that allows you to access the container registry. 
+
+<img src="../images/onpremise-dockerhub.png" width="500" /> 
+
+As explained above, Sigrid consists of several Docker containers. The container `sigrid-multi-analysis-import` will run directly from your development platform, all other containers are deployed to your Kubernetes cluster. These steps are explained in more detail in the following sections.
+
+### Updating Sigrid on-premise to a new version
```

Comment:
It is important that all components (APIs and analysis container) are update to the same version.

Reply:
Good one, will add.